node/subsystem-util/src/inclusion_emulator/staging.rs

```diff
@@ -0,0 +1,1448 @@
+// Copyright 2017-2022 Parity Technologies (UK) Ltd.
+// This file is part of Polkadot.
+
+// Polkadot is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Polkadot is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+//! The implementation of the inclusion emulator for the 'staging' runtime version.
+//!
+//! This is currently `v1` (`v2`?), but will evolve to `v3`.
+// TODO https://github.com/paritytech/polkadot/issues/4803
```

Comment:
Is this comment up to date? The issue has been closed.

Reply:
probably out of date, yeah.

Reply:
Added this to my docs edits pr